inc/plugin.class.php

```diff
@@ -541,7 +541,7 @@ function listPlugins() {
                      $function   = 'plugin_' . $plug['directory'] . '_check_prerequisites';
                      $do_install = $this->checkVersions($plug['directory']);
 
-                     if (function_exists($function)) {
+                     if ($do_install && function_exists($function)) {
```

Comment:
I guess this change is to fix something else ?

Reply:
Yes, this is not directly related to the issue :)